tests/roots/test-ext-autosummary-filename-map/index.rst

```diff
@@ -0,0 +1,13 @@
+
+:autolink:`autosummary_dummy_module.Foo`
+
+:autolink:`autosummary_importfail`
```

Comment:
This link is still remaining. Please remove this. I guess the link above is also not needed for this testcase.